### PR TITLE
CSPL-1310 Merge MC feature branch to develop

### PR DESCRIPTION
**Pull request by kashok-splunk**

### CODE REVIEW

File: test/smoke/smoke_test.go [hunk 1]
```diff
@@ -56,7 +56,7 @@ var _ = Describe("Smoke test", func() {
 			testenv.StandaloneReady(deployment, deployment.GetName(), standalone, testenvInstance)
 
 			// Verify MC Pod is Ready
-			testenv.MCPodReady(testenvInstance.GetName(), deployment)
+			// testenv.MCPodReady(testenvInstance.GetName(), deployment)
```
Comment (smohan-splunk) [inline test/smoke/smoke_test.go:59]: As clarified by @pdhanoyasplunk , we need to remove MC checks in the following as they are now redundant (as they are covered as part of MC testing):
* smartstore
* smoke
* scaling
To be taken up as part of another task

File: .circleci/config.yml.old [hunk 2]
```diff
@@ -48,7 +48,22 @@ workflows:
             - unit-tests
       - vulnerability-scan:
           requires:
-            - build-image     
+            - build-image
```
[the remark this answers is not in the document]
Comment (pdhanoyasplunk) [reply .circleci/config.yml.old:51]: No this is not required.

File: .github/workflows/int-test-workflow.yml [hunk 3]
```diff
@@ -4,11 +4,12 @@ on:
     branches:
       - develop
       - master
+      - feature-MC/CRD
```
[the remark this answers is not in the document]
Comment (pdhanoyasplunk) [reply .github/workflows/int-test-workflow.yml:7]: Yes we can remove it on our PR back to develop.

File: test/smoke/smoke_test.go (hunk 1 above)
Comment (pdhanoyasplunk) [reply test/smoke/smoke_test.go:59]: Task Created CSPL-1400

**Pull request merged**